src/default-controls.js

```diff
@@ -39,15 +39,64 @@ export class NextButton extends React.Component {
   constructor() {
     super(...arguments);
     this.handleClick = this.handleClick.bind(this);
+    this.nextButtonDisable = this.nextButtonDisabled.bind(this);
   }
   handleClick(event) {
     event.preventDefault();
     this.props.nextSlide();
   }
+
+  nextButtonDisabled(params) {
+    const {
+      wrapAround,
+      slidesToShow,
+      currentSlide,
+      cellAlign,
+      slideCount
+    } = params;
+
+    let disabledButton = false;
```

Comment:
similar to my comment above, I think this variable could be named `disableButton`. It follows the same convention as the function name. As a reader, it's more clear to think if we should "disable button?...yes (true) or no (false)" Does that make sense?

Reply:
I agree with Sarah that if this function were to be named `disableNextButton`, then this variable would read better as `disableButton`.  However, with the function named `nextButtonDisabled`, this variable would read better as `buttonDisabled`.  In both cases it's a shortened version of the method name (simply dropping "next"), and so it reads clearly that this is main boolean of the method.  And also both read better as questions like Sarah mentioned (should we 'disable button?", or is "button disabled?).

Reply:
I agree - leaving the method name as is, I still think this would be more clear as `buttonDisabled` signifying to the reader that the boolean value of this variable correlates to whether the button is disabled or not. This could just be a variance in how we read it, too